Foundation/NSCFString.swift

```diff
@@ -83,6 +87,10 @@ internal final class _NSCFConstantString : _NSCFString {
     required init(capacity: Int) {
         fatalError()
     }
+
+    required init(string aString: String) {
+        fatalError("init(string:) has not been implemented")
```

Comment:
the text for this probably should be "Constant strings cannot be constructed in code" or something along those lines